src/ngx_http_lua_shdict.c

```diff
@@ -375,6 +376,9 @@ ngx_http_lua_inject_shdict_api(ngx_http_lua_main_conf_t *lmcf, lua_State *L)
         lua_pushcfunction(L, ngx_http_lua_shdict_llen);
         lua_setfield(L, -2, "llen");
 
+        lua_pushcfunction(L, ngx_http_lua_shdict_lindex);
```

Comment:
I think we should only do the pure FFI implementation (through lua-resty-core) for such new shdict methods. We're retiring the CFunction implementation very soon. They'll be gone.

Reply:
Okay, I will rewrite this.